examples/mapviewer.py

```diff
@@ -189,29 +204,29 @@ def load_map_clicked(self, button):
         format = self.image_format_entry.get_text()
         if uri and format:
             if self.osm:
-                #remove old map
+                # remove old map
                 self.vbox.remove(self.osm)
```

Comment:
The sole fact that there is a comment indicates that we need to wrap the whole management of `self.vbox` into an object or helper functions. Such a trivial action like removing an object should really [be self-explanatory and not need a comment](https://softwareengineering.stackexchange.com/a/51313/23443).

Reply:
I didn't quite understand this. I removed the comment, as I believe the code to be clear enough.